### ISSUE TITLE
Achievements --> slack

### ISSUE THREAD

**Issue by fragmuffin** [fragmuffin/minecraft-ds-java#9]
Add optional notifications:

for example:

```
<username> has made the advancement [<achievement>]
```

Consider

- [x] advancements
- [x] kills
- [x] deaths

**Comment by fragmuffin** [comment]
Logs I can find.

**activity**

```
FraggyMuffin was slain by Zombie
FraggyMuffin drowned
FraggyMuffin has made the advancement [Monster Hunter]
FraggyMuffin lost connection: Disconnected
FraggyMuffin left the game
FraggyMuffin[/192.168.1.1:44964] logged in with entity id 2784 at ([world]-87.0274756623149, 70.0, 167.54542295469338)
```

**chat & commands?**

```
<FraggyMuffin> testing
FraggyMuffin issued server command: /dynmap fullrender
```

But I'm sure someone has a comprehensive list 🤷

**Comment by fragmuffin** [comment]
A seemingly full list of death messages is here : https://minecraft.fandom.com/wiki/Death_messages
They seem excessively varied... Wish they all had a common prefix or something :disappointed:

**Comment by fragmuffin** [comment]
ref: https://github.com/tjclement/minecraft-log-parser/blob/master/bin/mc_actions.py

**Comment by fragmuffin** [comment]
implemented in: #11
